src/lib.js

```diff
@@ -0,0 +1,331 @@
+export * from "./hamt/api.js"
+import * as Node from "./node.js"
+import { create as createBitmapIndexedNode } from "./node.js"
+import * as API from "./hamt/api.js"
+import * as Uint32Path from "./path/Uint32.js"
+import * as Uint8ArrayPath from "./path/Uint8Array.js"
+import { configure as configureBitField } from "./bitfield/lib.js"
+
+const NOT_FOUND = new RangeError("Not Found")
+
+/**
+ * A `bitWidth` determines the number of bits of the hash to use for index
+ * calculation at each level of the tree. In first layer will distribute
+ * children by the first `bitWidth` bits of the key hash. In the next layer
+ * next `bitWidth` number of bits of the key hash are used to determine
+ * placement of it's children and so on. Each node in the tree will hold
+ * `2 * bitWidth` number of elements.
```

Comment:
So if bitWidth is 5, each node has 10 elements? Or is it perhaps 32 elements i.e. `2 ** bitWidth`?

Reply:
Yeah it was typo I meant `2 ** bitWidth`